src/test/java/bio/terra/service/snapshotbuilder/query/table/PersonTest.java

```diff
@@ -0,0 +1,43 @@
+package bio.terra.service.snapshotbuilder.query.table;
+
+import static bio.terra.service.snapshotbuilder.utils.CriteriaQueryBuilderTest.assertQueryEquals;
+
+import bio.terra.common.category.Unit;
+import bio.terra.service.snapshotbuilder.query.FieldVariable;
+import bio.terra.service.snapshotbuilder.query.SqlRenderContext;
+import bio.terra.service.snapshotbuilder.query.SqlRenderContextProvider;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+
+@Tag(Unit.TAG)
+class PersonTest {
+
+  @ParameterizedTest
+  @ArgumentsSource(SqlRenderContextProvider.class)
+  void testAsPrimary(SqlRenderContext context) {
+    Person person = Person.forPrimary();
+    assertQueryEquals("person AS p", person.renderSQL(context));
+  }
+
+  @ParameterizedTest
+  @ArgumentsSource(SqlRenderContextProvider.class)
+  void testFromColumn(SqlRenderContext context) {
+    FieldVariable fieldVariable = Person.forPrimary().getColumn("column");
+    assertQueryEquals("p.column", fieldVariable.renderSQL(context));
+  }
+
+  @ParameterizedTest
+  @ArgumentsSource(SqlRenderContextProvider.class)
+  void testPersonId(SqlRenderContext context) {
+    FieldVariable fieldVariable = Person.forPrimary().personId();
+    assertQueryEquals("p.person_id", fieldVariable.renderSQL(context));
+  }
+
+  @ParameterizedTest
+  @ArgumentsSource(SqlRenderContextProvider.class)
+  void testCountPersonId(SqlRenderContext context) {
+    FieldVariable fieldVariable = Person.forPrimary().countPerson();
+    assertQueryEquals("COUNT(DISTINCT p.person_id)", fieldVariable.renderSQL(context));
```

Comment:
This is probably fine but it does duplicate testing that's done elsewhere. The only code to be tested in `Person` is
```
    super(sourceVariable);
    this.countPerson = sourceVariable.makeFieldVariable(PERSON_ID, "COUNT", null, true);
```
and the getter for `countPerson()`.

The duplication isn't good, but a bigger problem is that it makes this test depend on behavior of code outside `Person`. One way to fix this is to write the test using a mocked `SourceVariable` and not using `renderSql()`. For example

```
  @Test
  void testCountPersonId() {
    SourceVariable sourceVariable = mock(SourceVariable.class);
    FieldVariable count = new FieldVariable(null, null);
    when(sourceVariable.makeFieldVariable("person_id", "COUNT", null, true)).thenReturn(count);
    Person person = new Person(sourceVariable);
    assertThat(person.countPerson(), is(count));
  }
```

Right now we do use `renderSQL()` a lot so maybe this PR isn't the right time to start writing tests differently, but I think we should start thinking of how to get away from using it for all tests.